addons/hide-show-context-menu/userscript.js

```diff
@@ -0,0 +1,26 @@
+export default async function ({ addon, msg }) {
+  const vm = addon.tab.traps.vm;
+  const getSprite = (name) => vm.runtime.targets.find((target) => target.id === name);
+
+  const stateFromBool = (bool) => (bool ? "show" : "hide");
+  const sharedOptions = {
+    types: ["sprite"],
+    position: "assetContextMenuAfterDelete",
+    order: 9, // before folders - it's unpredictable how many slots folders takes up
+    border: true,
+  };
+
+  [true, false].forEach((visibility) => {
```

Comment:
This works but it's confusing.

Reply:
The alternative would be code duplicating, wouldn't it?